Description: Use libjs-jquery instead of embedded code
 Patch gitit to use the javascript linked from the libjs-jquery package,
 instead of the embedded one.
Author: Giovanni Mascellani <gio@debian.org>
Index: b/data/templates/expire.st
===================================================================
--- a/data/templates/expire.st
+++ b/data/templates/expire.st
@@ -1,5 +1,5 @@
 $if(usecache)$
-<script type="text/javascript" src="$base$/js/jquery.hotkeys-0.7.9.min.js"></script>
+<script type="text/javascript" src="$base$/js/jquery.hotkeys.js"></script>
 <script type="text/javascript">
 /* <![CDATA[ */
      \$(document).bind("keydown", "ctrl+r", function() {
Index: b/src/Network/Gitit/Layout.hs
===================================================================
--- a/src/Network/Gitit/Layout.hs
+++ b/src/Network/Gitit/Layout.hs
@@ -77,7 +77,7 @@ filledPageTemplate base' cfg layout html
                            'h':'t':'t':'p':_  -> x
                            _                  -> base' ++ "/js/" ++ x
 
-      scripts  = ["jquery-1.2.6.min.js", "jquery-ui-combined-1.6rc2.min.js", "footnotes.js"] ++ pgScripts layout
+      scripts  = ["jquery.min.js", "jquery-ui.min.js", "footnotes.js"] ++ pgScripts layout
       scriptLink x = script ! [src (prefixedScript x),
         thetype "text/javascript"] << noHtml
       javascriptlinks = renderHtmlFragment $ concatHtml $ map scriptLink scripts
Index: b/data/templates/page.st
===================================================================
--- a/data/templates/page.st
+++ b/data/templates/page.st
@@ -9,6 +9,7 @@
     <link href="$base$/_feed$pageUrl$" type="application/atom+xml" rel="alternate" title="$wikititle$ - $pagetitle$" />
     $endif$
     <title>$wikititle$ - $pagetitle$</title>
+    <link href="$base$/css/jquery-ui.min.css" rel="stylesheet" media="print" type= "text/css" />
     $if(printable)$
     <link href="$base$/css/print.css" rel="stylesheet" media="all" type= "text/css" />
     $else$
